packages/dht/test/end-to-end/Layer0Webrtc-Layer1.test.ts

```diff
@@ -28,22 +28,26 @@ describe('Layer 1 on Layer 0 with mocked connections', () => {
 
         const layer0Node1Id = createRandomDhtAddress()
         layer0Node1 = new DhtNode({
-            nodeId: layer0Node1Id
+            nodeId: layer0Node1Id,
+            entryPoints: [entrypointDescriptor]
```

Comment:
Are the explicit entry points needed because of the changes in this PR?

Reply:
Yes, previously connectivity checking was disabled in cases where a websocket server was configured. so the entry point list was not necessary in such cases

Reply:
Based on yesterday's discussion: these settings were changed to modify the test behavior so that nodeId is generated from the connectivity response. The `entryPoints` list is needed for making a connectivity a request as we use entrypoints as connectivity request targests. 

-> could remove redundant explicit nodeId as we get the id from connectivity response

Reply:
No need to remove here since explicit nodeId is still supported